README.md

```diff
@@ -5,3 +5,57 @@
 [![npm version](https://img.shields.io/npm/v/@rdfjs/parser-n3.svg)](https://www.npmjs.com/package/@rdfjs/parser-n3)
 
 N3 parser which implements the [RDFJS Sink interface](http://rdf.js.org/) using the [N3.js](https://github.com/rdfjs/N3.js) library.
+
+## Usage
+
+The package exports the parser as a class, so an instance must be created before it can be used.
+The `.import` method, as defined in the [RDFJS specification](http://rdf.js.org/#sink-interface), must be called to do the actual parsing.
+It expects a Turtle, N-Triples or N-Quads string stream.
+The method will return a stream which emits the parsed quads.
+It also emits `prefix` events as defined in the [RDFJS specification](http://rdf.js.org/#dom-stream-prefix).
+
+The constructor accepts the following options:
```

Comment:
I think it would be worth mentioning that it's a single `options` object with the following optional keys